src/interaction/command/circle.ts

```diff
@@ -0,0 +1,212 @@
+import Bot from "../../api/bot";
+import {
+  CategoryChannel,
+  ColorResolvable,
+  CommandInteraction,
+  OverwriteResolvable,
+} from "discord.js";
+import { settings } from "../../settings";
+import { CircleData } from "../../api/schema";
+import SlashCommand, {
+  SlashCommandContext,
+} from "../../api/interaction/slashcommand";
+
+export default class CircleCommand extends SlashCommand {
+  public constructor() {
+    super({
+      name: "circle",
+      description: "A suite of command that manage ACM Community Circles.",
+      permissions: [
+        { id: settings.roles.director, permission: true, type: "ROLE" },
+      ],
+    });
+    this.slashCommand.addSubcommand((subcommand) => {
```

Comment:
Originally i intended for this stuff to go in buildSlashCommand, but honestly this works with less bloat. Can you remove buildSlashCommand in this PR as well?

Reply:
Done 👍